dvc/repo/plots/__init__.py

```diff
@@ -196,7 +200,7 @@ def show(
             onerror=onerror,
             props=props,
         ):
-            _resolve_data_sources(data)
+            _resolve_data_sources(data, cache_remote_stream=True)
```

Comment:
`show()` is only used by DVC, not by Studio. So we only cache remote streams during `dvc plots show/diff` commands.